homeassistant/components/sector_alarm/__init__.py

```diff
@@ -0,0 +1,57 @@
+"""The Sector Alarm Integration."""
+import logging
+
+import sectoralarmlib.sector as sectorlib
+import voluptuous as vol
+
+from homeassistant.const import CONF_CODE, CONF_EMAIL, CONF_NAME, CONF_PASSWORD
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+
+DOMAIN = "sector_alarm"
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ID = "alarm_id"
```

Comment:
I actually meant to import CONF_ID from homeassistant.const and to remove this constant.